setup.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+
+# If a binary distribution exists for your OS and Python, congratulations.  You should
+# be able to simply install swig.
+#
+#
+# If you are doing a build from sources, you must also have have swig and a copy of
+# the cspice toolkit.
+#
+# [I need appropriate instructions here on how to get cspice.a and install it. so that
+#    <this top level directory>/cspice/lib/libcspice.a
```

Comment:
Actually the cspice distribution from NAIF calls the file `cspice/lib/cspice.a`, at least in the Linux version. Perhaps that was part of the problem I was having? The URL I gave earlier for downloading the Toolkit includes a pre-compiled version for each supported platform. You can just download each version, untar it, and see what's in it.

Reply:
This gets back to my original issue.  I have no way of knowing where the user is going to put their libcspice.a. And setup.py doesn't give me an easy way of looking for it on the user's machine.  That's why I decided the simplest solution was just to create it myself.